plugins/bigquery/dbt/adapters/bigquery/impl.py

```diff
@@ -595,6 +595,19 @@ def get_table_ref_from_relation(self, conn, relation):
                                           relation.identifier,
                                           conn)
 
+    def _update_column_dict(self, bq_column_dict, dbt_columns, parent=''):
+        # bq_colunmn_dict generated by SchemaField.to_api_repr()
+        dotted_column_name = '{}.{}'.format(parent, bq_column_dict['name']) if parent else bq_column_dict['name']
```

Comment:
Please split this up into a normal multiline if-else statement, this is too complicated for a one-liner.